docs/attestors/omnitrail.md

```diff
@@ -0,0 +1,148 @@
+## Schema
```

Comment:
We need to have docs for this new attestor. @fkautz maybe you have a couple mins to add this?

Reply:
Filed https://github.com/in-toto/witness/issues/465 to not block this PR.